.gitignore

```diff
@@ -4,3 +4,4 @@ dist/
 markdown.db
 *.tgz
 .markdowndb/
+.next/
```

Comment:
This is noise in the PR 😄 - is there any way to have this kind of tidying outside of the PR e.g. in a commit to main that you then merge to your PR?